src/main/java/com/revature/parasol/controllers/Force.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * 
+ */
+package com.revature.parasol.controllers;
+
```

Comment:
Created a Force class which should allow us to know definitively if the OAuth2 token is granted properly. This will create and print a string based on the information in the token. This string is a URL, the url we will call every time we want to query salesforce.